### PR TITLE
Change message on the 'vehicle tax' and 'check a vehicle is taxed' do…

### PR DESCRIPTION
**Pull request by jon-kirwan**
## What
https://trello.com/c/YoNZGyxT/1159-change-promotion-message-on-done-pages

Change the messages on the ‘vehicle tax’ and ‘check a vehicle is taxed’ done pages from 'Join the 14,000 new drivers every month who make the switch to an electric vehicle' to 'Find out how much money you can save on fuel by switching to an electric vehicle.’

## Why

The Go Ultra Low (GUL) website, which we currently link to from these done pages, is going to be taken down on 31st March 2022. The department want to instead link to Energy Savings Trust who they are now working closely with.

## Visual changes
<table role="table">
<thead>
<tr>
<th>Before (Give feedback on Tax your vehicle)</th>
<th>After</th>
</tr>
</thead>
<tbody>
<tr>
<td valign="top"><a target="_blank" rel="noopener noreferrer" href="https://user-images.githubusercontent.com/87758239/160423724-f5fa6228-880d-40d6-be83-552ac10542b6.png"></a></td>
<td valign="top"><a target="_blank" rel="noopener noreferrer" href="https://user-images.githubusercontent.com/87758239/160424155-b16e2a08-a929-40e4-8657-6e8e7af4c4b1.png"></a></td>
</tr>
</table>

<table role="table">
<thead>
<tr>
<th>Before (Give feedback on Check if a vehicle is taxed)</th>
<th>After</th>
</tr>
</thead>
<tbody>
<tr>
<td valign="top"><a target="_blank" rel="noopener noreferrer" href="https://user-images.githubusercontent.com/87758239/160423703-14a4e5c8-783c-4ea4-84af-f2657a4ddb43.png"></a></td>
<td valign="top"><a target="_blank" rel="noopener noreferrer" href="https://user-images.githubusercontent.com/87758239/160424135-bc66cf56-cbf3-491c-956a-036f38ba4954.png"></a></td>
</tr>
</table>

### CODE REVIEW

**Review by MartinJJones — APPROVED** [review]
Looks good, nice work Jon :)

File: config/locales/cy.yml [hunk 1]
```diff
@@ -202,7 +202,7 @@ cy:
       assistance_question: A gawsoch chi unrhyw gymorth i ddefnyddio'r gwasanaeth hwn heddiw?
       before_you_start: Cyn i chi ddechrau
       dissatisfied: Anfodlon
-      e_vehicle_info: Mae gwybodaeth ar gael am gerbydau trydan.
+      e_vehicle_info:
```
Comment (maxgds) [inline config/locales/cy.yml:205]: Did you add the new text to the Translations log (https://docs.google.com/spreadsheets/d/1Wips8w48mUoLE3UqY-TIeoHJcW9l371qUuRyCqbl5J0/edit#gid=0) ? We should capture that there's a need for new text whenever we lose a translation or create new strings so we can get the new wording when we next pay a translator.

**Comment by maxgds** [comment]
All looks good, I've taken the liberty of adding a label and updating your title as we need to be careful nobody helpfully merges these time based pieces of work once they're approved.

File: config/locales/cy.yml (hunk 1 above)
Comment (jon-kirwan) [reply config/locales/cy.yml:205]: > Looks good, nice work Jon :)

Great thanks Martin 👍

Comment (jon-kirwan) [reply config/locales/cy.yml:205]: > Did you add the new text to the Translations log (https://docs.google.com/spreadsheets/d/1Wips8w48mUoLE3UqY-TIeoHJcW9l371qUuRyCqbl5J0/edit#gid=0) ? We should capture that there's a need for new text whenever we lose a translation or create new strings so we can get the new wording when we next pay a translator.

Thanks Max, good to know, i've now updated the doc.

**Pull request merged**